proto/buf/alpha/registry/v1alpha1/display.proto

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/role.proto";
+
+// DisplayService is the display service that
+// provides information regarding to UI element display.
+service DisplayService {
+  // DisplayRepositoryRolesToManage returns which roles should be displayed
+  // to the user when they are managing contributors on the repository.
+  rpc DisplayRepositoryRolesToManage(DisplayRepositoryRolesToManageRequest) returns (DisplayRepositoryRolesToManageResponse);
```

Comment:
I don't know if we need to include `Display` in the RPC name since it's in the service. What do you think about
```suggestion
  // ListManagableRepositoryRoles returns which roles should be displayed
  // to the user when they are managing contributors on the repository.
  rpc ListManagableRepositoryRoles(ListManagableRepositoryRolesRequest) returns (ListManagableRepositoryRolesResponse);
```
If this endpoint is only used for adding users to a repository (in contrast to `DisplayRepositoryRolesToManageByUser`), maybe we can use a different word than "manage" too, maybe `ListAssignableRepositoryRoles`?

Reply:
I think our current practice is the RPC name should be self explainable without the service name, e.g. `RepositoryService.DeleteRepository`, `OrganizationService.DeleteOrganization`. 
I think `ListAssignableRepositoryRoles` is good and self explainable without service name, but for some endpoints later that only return a bool we may still want to put the `Display` in the RPC name, what do you think?
besides, this endpoint should apply to all add/update/remove action (cc @timostamm @seankimdev correct me if I am wrong), so I think `ListManageableRepositoryRoles` is good enough.

Reply:
> this endpoint should apply to all add/update/remove action

Yes, this should be the one that applies to the list of contributors, and does _not_ take into account that individual users may already have an implicit role on the repository.

Reply:
I think I agree that we can use `Display` as a prefix for `DisplayService.DisplayRepositorySettings` for example 👍🏻.